data/lib/core/game.lua

```diff
@@ -1,14 +1,14 @@
-function getGlobalStorageValueDB(key)
+function getGlobalStorage(key)
     local resultId = db.storeQuery("SELECT `value` FROM `global_storage` WHERE `key` = " .. key)
     if resultId ~= false then
-        local val = result.getString(resultId, "value")
+        local val = result.getNumber(resultId, "value")
```

Comment:
Try test this and send a feedback, please.
```suggestion
        local val
        if (result.getNumber(resultId, "value")) then
            val = result.getNumber(resultId, "value")
        else
            val = result.getString(resultId, "value")
        end
```